Review.md

```diff
@@ -3,10 +3,27 @@
 You can exercise your Google-Fu for this and any other _Sprint Challenge_ in the future.
 
 1.  Explain the difference between `RDBMS` and `SQL`.
+        RDBMS = Relational Database Management System, it is the system you use to structure the data in your database.
+        SQL = Structured Query Language, it is a language used to communicate with a database.
+
+
 1.  Why do tables need a `primary key`?
+        So that there can never be more than one, and so that you can reference the primary key as a foreign key in other tables.
```

Comment:
Tables need primary keys to differentiate the data. This helps uniquely identify each row, even if some of the data may be the same.